src/dashboard/src/components/administration/tests/test_storage.py

```diff
@@ -8,8 +8,8 @@
 THIS_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
-class TestAdministrationStorage(TestCase):
-    fixtures = [os.path.join(THIS_DIR, "fixtures", "test_user.json")]
+class TestStorage(TestCase):
+    fixtures = [os.path.join(THIS_DIR, "../../../../tests/fixtures", "test_user.json")]
```

Comment:
I don't know the prettiest way to do this (or a better way!), but I feel like we're more idiomatically using the following approach in other parts of the system:
```python
    dashboard_path = os.path.dirname(
        os.path.dirname(
            os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
        )
    )
    fixtures_path = os.path.join(
        THIS_DIR, dashboard_path, os.path.join("tests", "fixtures"), "test_user.json"
    )
    fixtures = [fixtures_path]
```
This should just work, but if we don't elect to change this now, we can always do it in future. Same for `test_usage.py` below.

Reply:
Hi @ross-spencer, I recently fixed the fixtures situation in https://github.com/artefactual/archivematica/commit/c63c1195e61b39cc53ad9f176059b3dae854b04e, I just didn't rebase this work after those changes. Will do and thanks for the remainder ;)

Reply:
Ah, I had a really loose memory of that @jraddaoui but couldn't get it back. Feels like such a long time ago! Anyway, looking forward to pushing on :slightly_smiling_face: